dependencies.gradle

```diff
@@ -8,6 +8,9 @@ dependencies {
     compileOnly("com.github.GTNewHorizons:Applied-Energistics-2-Unofficial:master-SNAPSHOT:dev") {
         transitive = false
     }
+    compileOnly("com.github.GTNewHorizons:Railcraft:master-SNAPSHOT:dev") {
```

Comment:
Railcraft provides an API artifact. Since you just removed the API you know it will suffice:
` compileOnly("com.github.GTNewHorizons:Railcraft:master-SNAPSHOT:api") {`